TOC-tidb-cloud.md

```diff
@@ -122,9 +122,8 @@
 - Tune Performance
   - [Overview](/tidb-cloud/tidb-cloud-tune-performance-overview.md)
   - Analyze Performance
-    - [Statement Analysis](/tidb-cloud/tune-performance.md)
-    - [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer)
```

Comment:
Is Key Visualizer a lesser used feature?

Reply:
![image](https://user-images.githubusercontent.com/79440533/192486229-a8cb423e-73fb-46ae-a373-8a4dbd3fd113.png)